### DISCUSSION TITLE
Export for database doesn't show buttons properly after successful procedure.

### DISCUSSION THREAD

**Discussion by ghost** [dbeaver/dbeaver#21274] [Ideas]
### Description

It would be nice to see an "OK" or "Close" button at the end of a successfully completed procedure.
Not just “<Back”, “Start”, “Cancel”. “Start” will start the dump process again. Does this mean that "Cancel" will delete the results of the dump procedure? (No, the dialog will close, and the results will not be affected) 

### DBeaver Version

Community Edition 23.2.0

### Operating System

Ubuntu/Windows

### Database and driver

MariaDB

### Steps to reproduce

1. Create a dump for existing database
2. After successfully completed procedure you will see enabled "< Back", "Start", "Cancel"

**Comment by ShadelessFox** [comment]
This behavior is consistent across all wizards in DBeaver. We don't think it should be changed. However, it can be further discussed here.

**Reply by ghost** [reply]
I am fine if pressing "Cancel" means Ok/Close here.

**Reply by ShadelessFox** [reply]
Yes, it just closes the wizard.

**Reply by ghost** [reply]
Ok. Thank you

**Comment by LonwoLonwo** [comment]
related: https://github.com/dbeaver/dbeaver/issues/17390
